docs/options.md

```diff
@@ -243,7 +243,7 @@ CIBW_BEFORE_ALL_LINUX: yum install -y libffi-dev
 
 A shell command to run before building the wheel. This option allows you to run a command in **each** Python environment before the `pip wheel` command. This is useful if you need to set up some dependency so it's available during the build.
 
-If dependencies are required to build your wheel (for example if you include a header from a Python module), set this to `pip install .`, and the dependencies will be installed automatically by pip. However, this means your package will be built twice - if your package takes a long time to build, you might wish to manually list the dependencies here instead.
+If dependencies are required to build your wheel (for example if you include a header from a Python module), you should add a pyproject.toml file instead of using this command. This is reproducable, and users who do not get your wheels (such as Alpine or ClearLinux users) will still benefit.
```

Comment:
Do we want to get rid of the other explanation (and warning!) completely?

Reply:
It’s still in the examples below, with a warning. But why would anyone want to do this? It’s horrible practice, and the real reason for it is our fault - we don’t allow setup_requires + install_requires to work because we don’t want to get an unrelated wheel. PyPA-build would solve this, by the way.

Reply:
Fair enough. Just worth checking, given that there's a balance in `cibuildwheel` between educating/nudging towards good practices and providing useful information for common goals.